admin-dev/themes/new-theme/js/pages/order/view/order-invoices-refresher.js

```diff
@@ -51,6 +51,7 @@ export default class OrderInvoicesRefresher {
 
           $existingInvoicesGroup.append(`<option value="${invoiceId}">${invoiceName}</option>`);
           $paymentInvoiceSelect.append(`<option value="${invoiceId}">${invoiceName}</option>`);
+          $appProductInvoiceSelect.append(`<option value="${invoiceId}">${invoiceName}</option>`);
```

Comment:
You need to empty it before adding the options or you will have doublons (like for the other selectors before the loop)

Reply:
done